tests/test_helpers.py

```diff
@@ -1,11 +1,17 @@
 from django.conf import settings
 
 from cms.test_utils.testcases import CMSTestCase
+from cms.test_utils.util.fuzzy_int import FuzzyInt
 
 from djangocms_navigation.helpers import get_navigation_node_for_content_object
 from djangocms_navigation.test_utils import factories
 from djangocms_navigation.test_utils.polls.models import Poll, PollContent
 
+try:
+    from djangocms_versioning.constants import PUBLISHED
```

Comment:
Versioning is very much an integral part of Navigation, there's no need for this try catch.

Reply:
Done updated